spring-integration-ftp/src/main/java/org/springframework/integration/ftp/outbound/FtpMessageHandler.java

```diff
@@ -48,10 +48,11 @@ public FtpMessageHandler(FtpRemoteFileTemplate remoteFileTemplate) {
 		super(remoteFileTemplate);
 	}
 
-	public FtpMessageHandler(RemoteFileTemplate<FTPFile> remoteFileTemplate, FileExistsMode mode) {
-		super(remoteFileTemplate, mode);
+	public FtpMessageHandler(FtpRemoteFileTemplate template, FileExistsMode fileExistsMode) {
```

Comment:
That's not correct to make such a breaking change.

We need to introduce a new ctor (to align with an `SftpMessageHandler`) and deprecate this one.

Reply:
I restored it back as i was before